integrationtests/security-it/pom.xml

```diff
@@ -83,13 +83,7 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-failsafe-plugin</artifactId>
             <configuration>
-               <!-- Force use of JUnit, since TestNG+Arquillian break in wonderful ways -->
-               <testNGArtifactName>none:none</testNGArtifactName>
-               <forkCount>1</forkCount>
-               <reuseForks>false</reuseForks>
-               <properties>
-                  <listener>${junitListener}</listener>
-               </properties>
+               <forkMode>once</forkMode>
```

Comment:
`forkMode` is [deprecated](http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#forkMode), and are you sure we don't need to force the use of JUnit any more?